devicetype.go

```diff
@@ -0,0 +1,72 @@
+package balena
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"go.einride.tech/balena/odata"
+)
+
+const deviceTypeBasePath = "v6/device_type"
+
+// DeviceService handles communication with the device related methods of the
```

Comment:
```suggestion
// DeviceTypeService handles communication with the device type related methods of the
```

Reply:
Thanks for catching that. It's a silly copy-paste error

Reply:
Did you miss the second suggestion in your fix?

Reply:
Not that I was aware of. Though I did find other references to UUIDs in the documentation. I've fixed those now as well.

Reply:
I meant in this suggestion. There were 2 suggested changes in this comment